cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureTerminationHelperService.java

```diff
@@ -30,6 +30,8 @@ public class AzureTerminationHelperService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AzureTerminationHelperService.class);
 
+    private static final String LOAD_BALANCER_IP_NAME = "loadbalancer";
```

Comment:
where is this coming from?

Reply:
This constant is being used to filter the public IPs resources related to LB that cannot be deleted. 
The LB IP's name starts with "loadbalancer".

Reply:
why does it start with it? are we naming it this way? is it a provider constraint?

Reply:
The public IP of the load balancer is created by the provider but it follows the name of the Load balancer. For the Load balancer we have this prefix "LoadBalancer".

Reply:
Make sense, it's done. Thanks